examples/nabu/inputtest.c

```diff
@@ -0,0 +1,122 @@
+#include <conio.h>
+#include <games.h>
+#include <arch/nabu/paddles.h>
+#include <sys/ioctl.h>
+
+#include "font.h"
+
+#define MODE_TEXT 0
+
+// decode and print joystick byte
+void printJoystick(uint8_t b)
+{
+	char state[6];
+
+	state[0] = '.';
+	state[1] = '.';
+	state[2] = '.';
+	state[3] = '.';
+	state[4] = '.';
+	state[5] = 0;
+
+	if (b & MOVE_LEFT) {
+		state[0] = 'L';
+	}
+	if (b & MOVE_DOWN) {
+		state[1] = 'D';
+	}
+	if (b & MOVE_RIGHT) {
+		state[2] = 'R';
+	}
+	if (b & MOVE_UP) {
+		state[3] = 'U';
+	}
+	if (b & MOVE_FIRE) {
+		state[4] = 'F';
+	}
+	printf("%s", state);
+}
+
+void main() {
+
+	uint8_t mode = MODE_TEXT;
+	void *font_ptr = &font_6x8_ascii;
```

Comment:
Do you mind using either `font_8x8_einstein_system` or `font_8x8_msx_system`? Both of them should work correctly in text mode without being cut off.

I'll add a 6x8 category after the next release to avoid having to guess which ones should work.